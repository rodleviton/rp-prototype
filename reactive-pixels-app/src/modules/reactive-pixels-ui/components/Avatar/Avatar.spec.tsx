import * as React from 'react';
import { render } from 'react-testing-library';
import WithTheme from '../../utils/test/WithTheme';
import { Avatar } from './';

describe('<Avatar />', () => {
  it('should render <Avatar /> component', () => {
    const { container } = render(
      <WithTheme>
        <Avatar />
      </WithTheme>,
    );

    expect(container.firstChild).toMatchSnapshot();
  });
});
