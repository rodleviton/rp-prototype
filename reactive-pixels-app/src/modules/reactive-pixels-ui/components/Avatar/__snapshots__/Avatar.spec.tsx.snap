// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Avatar /> should render <Avatar /> component 1`] = `
<div
  class="css-4s191h css-otakqs"
>
  <img
    class="css-1vm9b5l css-q7lffx"
  />
</div>
`;
